dashboard/src/components/ServiceInstanceView/ServiceInstanceStatus.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from "react";
+
+import { AlertTriangle } from "react-feather";
+import { IServiceInstance } from "shared/ServiceInstance";
+import Check from "../../icons/Check";
+import Compass from "../../icons/Compass";
+import "../DeploymentStatus/DeploymentStatus.css";
+
+interface IServiceInstanceStatusProps {
+  instance: IServiceInstance;
+}
+
+class ServiceInstanceStatus extends React.Component<IServiceInstanceStatusProps> {
```

Comment:
this can be a stateless component as well

Reply:
:+1: